serverv2/cometbft/params.go

```diff
@@ -0,0 +1,26 @@
+package cometbft
+
+import (
+	"context"
+
+	cmtproto "github.com/cometbft/cometbft/proto/tendermint/types"
+)
+
+const InitialAppVersion uint64 = 0
+
+// ParamStore defines the interface the parameter store used by the BaseApp must
+// fulfill.
+type ParamStore interface {
+	Get(ctx context.Context) (cmtproto.ConsensusParams, error)
+	Has(ctx context.Context) (bool, error)
+	Set(ctx context.Context, cp cmtproto.ConsensusParams) error
+}
```

Comment:
```suggestion
type ParamStore interface {
	Get(ctx context.Context) (cmtproto.ConsensusParams, error)
}
```

this only really needs Get, because on init we would pass in items needed to start the chain.